index.html

```diff
@@ -42,7 +42,8 @@ <h5>Language</h5>
                         </article>
                         <article class="frameworks">
                             <h5>Frameworks</h5>
-                            <ul class="top">                                <li>Bootstrap</li>
+                            <ul class="top">
```

Comment:
- [ ] You have done well working on the desktop version. However, I noticed horizontal scrolling in the desktop version. It will be nice if you could fix it. 
See test demo of your desktop version

DEMO|
:--------:|
![a1](https://user-images.githubusercontent.com/39523312/159910405-743196ee-a286-4b57-8d64-ec9f589f16bc.gif)